src/NetAnalyzers/CSharp/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,9 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+### New Rules
```

Comment:
The code generating this markdown file was fixed before to add empty lines and adhere to markdownlint rules.
The code reading the file doesn't care about blank lines, so this should be okay.

```suggestion

### New Rules

```